client/package.json

```diff
@@ -4,6 +4,7 @@
 	"private": true,
 	"scripts": {
 		"build": "react-scripts build",
+		"ci": "tsc && pnpm test",
```

Comment:
why we need this ?

Reply:
Because it's cool to be able to quickly execute in local the things that are checked in the ci, so we can avoid backs and froths and wait for ci checks on github actions. It's just to save time and anticipate ci problems